src/audio/module_adapter/module/generic.c

```diff
@@ -225,12 +228,13 @@ int module_prepare(struct processing_module *mod)
 	return ret;
 }
 
-int module_process(struct processing_module *mod, struct input_stream_buffer *input_buffers,
-		   int num_input_buffers, struct output_stream_buffer *output_buffers,
-		   int num_output_buffers)
+int module_process_stream_buffer(struct processing_module *mod,
```

Comment:
I decode methods that use input_stream_buffer and output_stream_buffer here
Inside of those there are 2 ways of processing - audio stream and raw data

**I don't like it at all** - those are structures from IADK and should never be used here. Anyway - backward compatibilty 

Anyway - you're right that it may sound too close to audio_stream or something
Changing to module_process_legacy